Duplicati/WebserverCore/Endpoints/V1/Backups.cs

```diff
@@ -161,7 +162,7 @@ private static Dto.CreateBackupDto ExecuteAdd(Connection connection, Dto.BackupA
                 Name = data.Backup.Name,
                 Description = data.Backup.Description,
                 Tags = data.Backup.Tags,
-                TargetURL = data.Backup.TargetURL,
+                TargetURL = EncryptedFieldHelper.Encrypt( data.Backup.TargetURL),
```

Comment:
Is this required? Would the `Connection` helper not encrypt this when saving?
I think it reduces complexity if only the database interaction is aware of the encryption, so anything in/out of `Connection` appears unencrypted.

Reply:
Yes, it is required. As explained [here](https://github.com/duplicati/duplicati/pull/5466#discussion_r1721817614) in the way it was implemented the TargetURL field of backup table is handled outside of Connection.